handsontable/src/core.js

```diff
@@ -3350,11 +3350,13 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
    * @function getCellMeta
    * @param {number} row Visual row index.
    * @param {number} column Visual column index.
+   * @param {object} options Execution options for the `getCellMeta` method.
+   * @param {boolean} [options.skipMetaExtension] If `true`, skips extending the cell meta object. Default: `false`.
    * @returns {object} The cell properties object.
    * @fires Hooks#beforeGetCellMeta
    * @fires Hooks#afterGetCellMeta
    */
-  this.getCellMeta = function(row, column) {
+  this.getCellMeta = function(row, column, options = { skipMetaExtension: false }) {
```

Comment:
Can I ask you to cover the new argument with tests? I was thinking about 2 tests, one called without `skipMetaExtension` defined, which checks that `calls`, `beforeGetCellMeta`, and `afterGetCellMeta` are called and the other with `skipMetaExtension: true`.

Reply:
Many thanks